index.html

```diff
@@ -51,16 +63,69 @@ <h2 id="project_tagline"></h2>
 <h2>
 <a id="how-to-install" class="anchor" href="#how-to-install" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>How to Install?</h2>
 
-<p>Binary is relased via <a href="https://jitpack.io/#Ericsson/eiffel-remrem-publish/">jitPack</a></p>
+<p>Binary is relased via <a href="https://jitpack.io/#Ericsson/eiffel-remrem-publish">jitPack common</a></p>
+<p><b>Download the latest REMREM Publish CLI Binary via </b><a onClick="getLatestVersion('cli','jar');">publish-cli.jar</a></p>
+<p><b>Download the latest REMREM Publish Service Binary via </b><a onClick="getLatestVersion('service','war');">publish-service.war</a></p>
+<div id="downloadLink" style="visibility: hidden;color: red;font-size: 14px;">
+</div>
+<script type="text/javascript">
+function getLatestVersion(type,extn){
+	var xhr = new XMLHttpRequest(); 
+	xhr.open("GET", "https://jitpack.io/v/Ericsson/eiffel-remrem-publish.svg"); 
+ 	xhr.responseType = "blob";//force the HTTP response, response-type header to be blob
+ 	xhr.onload = function() 
+ 	{
+ 		var blob = xhr.response;
+ 		var reader = new FileReader();
+ 		reader.addEventListener("loadend", function() {
+			var result = reader.result;
+			if(result=="Repo not found or no access token provided"){
+				document.getElementById("downloadLink").innerHTML = result;
+				document.getElementById("downloadLink").style.visibility = "";
+			}
+			else{
+				document.getElementById("downloadLink").innerHTML = result;
+				var val = document.getElementsByTagName("text")[2].innerHTML;
+				val = val.trim();
+				urlLink = "https://jitpack.io/com/github/Ericsson/eiffel-remrem-publish/publish-"+type +"\/"+val+"/publish-"+type +"-"+val+"."+extn;
+				var link = document.createElement("a");
+				link.setAttribute('id',"MYLINK");
+				link.href = urlLink;
+				document.body.appendChild(link);
+                link.setAttribute("type", "hidden");
+				link.click();
+				
+				
+				}
+	 	});
+ 		reader.readAsBinaryString(blob);
+ 	}
+ 	xhr.send();
+}
+</script>
+
+<h3>
+<a id="installation" class="anchor" href="#installation" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Installation (Micro service)</h3>
+
+<p>Start the self executing Publish-cli JAR</p>
+<pre>java -jar publish-cli.jar</pre>
+
+<p>Deploy the self executing Publish-Service WAR in Tomcat to use REST services.</p>
```

Comment:
I guess this should be moved to the bottom of the page, where we are describing the Web application.